app/assets/stylesheets/z_custom.scss

```diff
@@ -0,0 +1,302 @@
+/* COLUMNS */
+
+.column, .columns{
+  padding: 20px 0px;
+}
+.container.staticpages {
+  max-width: 100%;
+}
+
+.inner-container{
+  max-width: 1200px;
+  margin: 0 auto;
+}
+/* FONTS */
+h1, h2, h3, h4, h5, h6, .main-nav{
+  font-family: 'Oswald', sans-serif;
+  color:#fff;
+  text-transform:uppercase;
+  letter-spacing:.006em;
+  text-shadow: 1px 1px 2px rgba(0, 0, 0, 1);
+}
+h1, h2, h6{
+  font-family:'Arvo', serif;
+  font-weight:bold;
+  letter-spacing:.006em;
+}
+h6{
+  text-align:center;
+  min-height:50px;
+}
+
+h5{
+  text-align:center;
+}
+a{
+  text-decoration:none;
+  color:#fff;
+}
+
+a:hover{
+  color:#8a1f1a;
+}
+
+p, table{
+  color:#fff;
+  text-shadow: 1px 1px 2px rgba(0, 0, 0, 1);
+}
+
+
+.contact h1, .about h5{
+  float:right;
+}
+
+hr{
+  border: 5px solid #fff;
+}
+
+/* NAV */
+
+
+.top-nav ul li, .main-nav ul li {
+  display:inline-block;
+  float:left;
+}
+
+.top-nav ul li{
+  float:right;
+  padding:0px 10px;
+}
+
+.main-nav{
+  background:url(image_path("nav-ribbon.png")) no-repeat;
+  background-size: 100% 110px;
+  font-size: 30px;
+  min-height: 130px;
+  // margin: -70px auto 0;
+}
+
+.main-nav ul{
+  margin-left:80px;
+}
+.main-nav ul li{
+  width:20%;
+  padding-top:30px;
+}
+blockquote{
+  text-align:center;
+  color:#fff;
+  font-style:italic;
+  position:relative;
+
+  margin-top:35px;
+}
+
+
+blockquote::before{
+  content: "";
+  background: url(image_path("wheat-left.png")) no-repeat;
+  display: inline-block;
+  min-height: 100px;
+  min-width: 110px;
+  position: absolute;
+  left: -95px;
+  top: -30px;
+}
+
+blockquote::after{
+  content: "";
+  background: url(image_path("wheat-right.png")) no-repeat;
+  display: inline-block;
+  min-height: 100px;
+  min-width: 110px;
+  position: absolute;
+  right: -95px;
+  top: -30px;
+}
+
+.logo{
+  margin:-62px 0 0 -25px;
+
+}
+.logo img{
+  width: 135px;
+  height: 188px;
+}
+
+/*BEERS*/
+
+/* FOOT */
+
+/* HOME */
+
+img{
+  width:100%;
+}
+
+body{
+  background:url(image_path("chalk.jpg")) repeat;
+  color:#fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space